packages/svelte-ux/src/lib/stores/formStore.ts

```diff
@@ -62,12 +62,14 @@ export default function formStore<T extends Objectish = any>(
         }
       }
 
-      const newState = finishDraft(draft, (patches, inverseChanges) => {
-        undoList.push(inverseChanges);
-      }) as T;
+      if (!options?.isFormSubmission) {
```

Comment:
I'm trying to think this through, but do think not updating the local state when submitting the form is needed (or even desired).  It seems like it could flash an optimistic view while the form submission in inflight, but not sure if that's truly possible, or unwanted, but I could completely be overlooking something as well.

If we don't need this, it greatly simplifies everything as we only need to remove the early `return true` I added in the previous PR (I completely overlooked the later `return true` at the bottom of `commit()`.

Thoughts?

Reply:
I also don't think that this would be an issue. I thought you added the early `return true` for performance reasons, but I also don't see a problem if we just update the store even though it might be not necessary.